recipes/org-clock-reminder

```diff
@@ -0,0 +1,4 @@
+(org-clock-reminder
+ :fetcher github
+ :repo "inickey/org-clock-reminder"
+ :files ("*.el" "icons/*.png"))
```

Comment:
Using `:defaults` is better here, since it will automatically exclude `-test.el` files if you wanted to add some in the future.
```suggestion
 :files (:defaults "icons/*.png"))
```